node_modules/gh-orgunit/lib/internal/dao.js

```diff
@@ -483,7 +483,15 @@ var exportOrgUnit = module.exports.exportOrgUnit = function(orgUnit, format, cal
                 {'model': DB.Serie, 'include': [
                     {'model': DB.Event, 'include': [
                         {'model': DB.User, 'as': 'Organisers'}
-                    ]}
+                    ]},
+
+                    // Include the first organisational unit (=A) the series was added to. If this organisational unit (A) doesn't
```

Comment:
This comment is indicating a single org unit would be returned, but isn't it multiple due to the `has many`? I'm hoping you can figure out how to rewrite this comment based on the confusions I'm having...